src/init/inst-functions.sh

```diff
@@ -689,16 +689,28 @@ InstallCommon()
     INSTALL="install"
 
     if [ ${INSTYPE} = 'server' ]; then
-        OSSEC_CONTROL_SRC='./init/wazuh-server.sh'
+        OSSEC_CONTROL_SRC_TEMPLATE='./init/wazuh-server.sh'
         OSSEC_CONF_SRC='../etc/ossec-server.conf'
     elif [ ${INSTYPE} = 'agent' ]; then
-        OSSEC_CONTROL_SRC='./init/wazuh-client.sh'
+        OSSEC_CONTROL_SRC_TEMPLATE='./init/wazuh-client.sh'
         OSSEC_CONF_SRC='../etc/ossec-agent.conf'
     elif [ ${INSTYPE} = 'local' ]; then
-        OSSEC_CONTROL_SRC='./init/wazuh-local.sh'
+        OSSEC_CONTROL_SRC_TEMPLATE='./init/wazuh-local.sh'
         OSSEC_CONF_SRC='../etc/ossec-local.conf'
     fi
 
+    # Reading from files again to prevent an error during upgrade
```

Comment:
I think we can avoid the reload of VERSION if we update the version inside wazuh_control during the execution of bump_version script.
In addition, we can eliminate DATE to fully avoid the sed in this step.